src/frontend/context/AppContext.ts

```diff
@@ -287,7 +292,11 @@ const [AppProvider, useApp] = createUseContext(() => {
                     getName(flettefelter.land, spesifikkLocale ?? valgtLocale) ?? flettefelter.land
                 );
             case ESanityFlettefeltverdi.YTELSE:
-                throw Error('Flettefeltet YTELSE er ikke støttet enda');
+                return plainTekst(
+                    tekster()[ESanitySteg.FELLES].frittståendeOrd.barnetrygd,
+                    undefined,
+                    spesifikkLocale ?? valgtLocale
```

Comment:
Jeg er litt usikker på om vi egentlig trenger `spesifikkLocale` i hele denne funksjonen. Når vil vi egentlig overstyre locale som er valgt for hele appen?